package.json

```diff
@@ -0,0 +1,34 @@
+{
+    "name": "arches_her",
+    "license": "AGPL-3.0-only",
+    "scripts": {
+        "build_development": "npm run eslint:check && npm run ts:check && cross-env NODE_OPTIONS=--max-old-space-size=2048 webpack --config ./webpack/webpack.config.dev.js",
+        "build_production": "npm run eslint:check && npm run ts:check && cross-env NODE_OPTIONS=--max-old-space-size=2048 NODE_ENV=production webpack --config ./webpack/webpack.config.prod.js",
+        "build_test": "npm run eslint:check && npm run ts:check && cross-env NODE_OPTIONS=--max-old-space-size=2048 webpack --config ./webpack/webpack.config.dev.js --env test=true",
+        "eslint:check": "eslint **/src",
+        "eslint:fix": "eslint **/src --fix",
+        "eslint:watch": "nodemon --watch . --ext ts,vue --exec npm run --silent eslint:check",
+        "gettext:extract": "vue-gettext-extract",
+        "gettext:compile": "vue-gettext-compile",
+        "prettier:check": "prettier {{ project_name }}/src --check",
+        "prettier:fix": "prettier {{ project_name }}/src --write",
```

Comment:
Hey @chiatt. This has to be manually fixed before pasting. I guess we should update the release note to call this out? Oversight on my part.